app/lib/exchange-rate/exchange-rate-service.ts

```diff
@@ -27,7 +29,19 @@ class ExchangeRateService {
 
     for (const provider of providersForTickers) {
       try {
-        return await provider.getRates({ tickers, signal });
+        const rates = await provider.getRates({ tickers, signal });
+
+        // Add inverse rates
+        for (const ticker of tickers) {
+          const [from, to] = ticker.split('-');
+          const rate = rates[ticker];
+          const inverseTicker = `${to}-${from}` as const;
+          if (!(inverseTicker in rates)) {
+            rates[inverseTicker] = new Big(1).div(rate).toNumber();
```

Comment:
I tried for a minute to make the type of rates:
```ts
export type Rates = {
  timestamp: number;
  [ticker: Ticker]: NumberInput; // NumberInput is from money.ts; changed type from number
};
```

so that we could set the value here as a Big, but I was having some trouble parsing the value in the react query. The issue was around the Big value just being bytes by the time it got returned by the query function. 

Is it fine to just call `toNumber` here? I can spend some more time, I didn't try for long, if you think that its important to not convert to a number here

Reply:
number can safely (without rounding) store numbers up to 15 decimal places. try to write `1.0004251807017989` to console and it will get rounded to `1.000425180701799`. not sure if this is what we want especially given that coinbase api sometimes does return the numbers with 16 decimals.

maybe we should have rates returned by the providers as `Big` and then here too we could leave it as Big

Reply:
to add I don't think it makes sense to have it as `NumberInput` but it does as `Big`

Reply:
fixed with #235